tests/integration/test_remote_write_grafana_agent.py

```diff
@@ -47,6 +48,60 @@ async def test_remote_write_with_grafana_agent(ops_test, prometheus_charm):
     )
 
 
+@pytest.mark.abort_on_fail
+async def test_check_data_persist_on_kubectl_delete_pod(ops_test, prometheus_charm):
```

Comment:
Is this in addition to or instead of `test_kubectl_delete.py`?

Reply:
This test may completely replace `test_kubectl_delete.py`, since do the same and also check for data persistance